.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,35 @@
 Issue: #
 
-## What Changed
+## Summary
 
-<!-- Insert a description below. -->
+<!--
+  Describe the changes made.
+  What would someone unfamiliar with the project (but familiar with the codebase) need to understand what happened and why?
+-->
 
-## How to test
+## How to QA
 
-<!-- Add an explanation below for the reviewers to help them test your changes. -->
+<!--
+  Provide step-by-step instructions on how to verify the work.
+  Use the canary build of the e2e packages (created via the `release_canary` CI job with this PR)
+  If necessary, include details on how to test in production.
+
+  Don't forget to test related features!
+-->
+
+1.
+
+Check ☑️ the appropriate box once you've finished QA:
+
+- [ ] Author - QA OK
+- [ ] Reviewer - QA OK
+
+<!-- ignore-task-list-start -->
+
+<!--
+## Release checklist
+- [ ] [Changeset](https://github.com/chromaui/chromatic-e2e/blob/main/DEVELOPMENT.md#pr-workflow-with-changesets) created in this PR
+
+-->
+
+<!-- ignore-task-list-end -->
```

Comment:
Do we actually want this ignored? When would we not create a Changeset?

Reply:
Good point! I'll have it commented-in with a note to remove if not publishing to NPM.